test/integration/relay/connection.test.js

```diff
@@ -847,11 +847,9 @@ describe('relay', function () {
 
       expect(sqlSpy.callCount).to.equal(1);
 
-      const nodeNames = result.data.user.projects.edges.map(edge => {
-        return edge.node.tasks.edges.map(edge => {
-          return edge.node.name
-        }).sort()
-      });
+      const nodeNames = result.data.user.projects.edges.map(
+        ({node: {tasks: {edges}}}) => edges.map(({node: {name}}) => name).sort()
+      );
```

Comment:
Previous code was in my opinion far easier to read. I'm not a huge fan of greatly nested arrow functions